packages/framework/client-logger/fluid-telemetry/package.json

```diff
@@ -77,10 +77,20 @@
 		"format:prettier": "prettier --write . --cache --ignore-path ../../../../.prettierignore",
 		"lint": "fluid-build . --task lint",
 		"lint:fix": "fluid-build . --task eslint:fix --task format",
-		"test": "npm run test:mocha",
-		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
-		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.*js\" --exit",
-		"test:mocha:esm": "mocha --recursive \"lib/test/**/*.spec.*js\" --exit",
+		"start:tinylicious:test": "tinylicious > tinylicious.log 2>&1",
+		"test": "start-server-and-test start:tinylicious:test 7070 test:mocha:all",
```

Comment:
Nit: this works, and might be more performant, but it diverges pattern-wise from how CI runs the tests (as separate steps). It might be helpful to be consistent with CI and just run the standard mocha tests and the real service ones separately via `concurrently npm:test:mocha:unit npm:test:mocha:end-to-end`.